src/doc/yaml/enrollment.yaml

```diff
@@ -0,0 +1,652 @@
+
+# (c) Secure Identity Alliance
+
+openapi: 3.0.0
+info:
+  description: OSIA Enrollment Interface
+  version: 1.0.0
+  title: OSIA Enrollment Interface
+  license:
+    name: SIA
+    url: "https://raw.githubusercontent.com/SecureIdentityAlliance/osia/master/LICENSE"
+tags:
+  - name: Enrollment
+servers:
+  - url: http://enrollment.com/
+  - url: https://enrollment.com/
+paths:
+
+  /v1/enrollment/{enrollmentId}:
+    post:
+      tags:
+        - Enrollment
+      summary: Create one enrollment
+      operationId: createEnrollment
+      parameters:
+        - name: enrollmentId
+          in: path
+          description: the id of the enrollment
+          required: true
+          schema:
+            type: string
+        - name: finalize
+          in: query
+          description: Flag to indicate that data was collected
+          required: true
```

Comment:
Should be optional maybe? (default to false)

Reply:
> Should be optional maybe? (default to false)
what is the property for default?

Reply:
I don't think we can define the default value in OpenAPI. This would have to be added to the description.